server/routerlicious/packages/services-client/package.json

```diff
@@ -56,6 +56,7 @@
     "@fluidframework/protocol-base": "^0.1030.0",
     "@fluidframework/protocol-definitions": "^0.1025.0-0",
     "axios": "^0.21.1",
+    "crc-32": "1.2.0",
```

Comment:
I know the the hash function is still a work in progress, but at some point we may want to use a different module for hashing in node.js
For example http://engineering.voxer.com/2013/12/02/hardwareaccelerated-crc32-for-nodejs/ / https://github.com/Voxer/sse4_crc32 uses a native node module for faster crc32's. Just something to keep in mind

Reply:
Yes this one is just to get the ball rolling. Let me know if you prefer any specific one